### PR TITLE
updated images for Rust to ghcr

### PR DESCRIPTION
**Pull request by gOOvER**
# Description

- updated both eggs to PTDL_v2
- updated images to ghcr
- updated installoscript to latest
- updated installimage

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?

### CODE REVIEW

File: game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json [hunk 1]
```diff
@@ -24,8 +24,8 @@
...
...] && printf %s '+@sSteamCmdForcePlatformType windows' ) +app_update ${SRCDS_APPID} $( [[ -z ${SRCDS_BETAID} ]] || printf %s \"-beta ${SRCDS_BETAID}\" ) $( [[ -z ${SRCDS_BETAPASS} ]] || printf %s \"-betapassword ${SRCDS_BETAPASS}\" ) ${INSTALL_FLAGS} validate +quit ## other flags may be needed depending on install. looking at you cs 1.6\r\n\r\n## set up 32 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so\r\n\r\n## set up 64 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk64\r\ncp -v linux64\/steamclient.so ..\/.steam\/sdk64\/steamclient.so\r\n\r\n## add below your custom commands if needed\r\n\r\n\r\nif [ ${REGEN_SERVER} == \"1\" ]; then\r\n    cd \/mnt\/server\/\r\n    rm -rf ${REMOVE_FILES}\r\nfi\r\n\r\nif [ $WORLD_SEED == \"0\" ]; then\r\n    if [ ! -f \/mnt\/server\/seed.txt ]; then\r\n        rm -sf \/mnt\/server\/seed.txt\r\n    fi\r\n    \r\n    cat \/dev\/urandom | tr -dc '1-9' | fold -w 5 | head -n 1 > \/mnt\/server\/seed.txt\r\nfi\r\n\r\n## install end\r\necho \"-----------------------------------------\"\r\necho \"Installation completed...\"\r\necho \"-----------------------------------------\"",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json]: the apt install and apt update are not needed with this image

File: game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json [hunk 2]
```diff
@@ -127,7 +137,8 @@
             "default_value": "CHANGEME",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json:137]: this should not have a default value

File: game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json [hunk 3]
```diff
@@ -163,7 +177,18 @@
...
+        {
+            "name": "SRCDS_APPID",
+            "description": "",
+            "env_variable": "SRCDS_APPID",
+            "default_value": "258550",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|max:20",
+            "field_type": "text"
         }
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json:192]: where is the auto update variable?

File: game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json [hunk 4]
```diff
@@ -1,30 +1,31 @@
...
...mCMD fails otherwise for some reason, even running as root.\r\n# This is changed at the end of the install process anyways.\r\nchown -R root:root \/mnt\r\nexport HOME=\/mnt\/server\r\n\r\n## install game using steamcmd\r\n.\/steamcmd.sh +force_install_dir \/mnt\/server +login ${STEAM_USER} ${STEAM_PASS} ${STEAM_AUTH} $( [[ \"${WINDOWS_INSTALL}\" == \"1\" ]] && printf %s '+@sSteamCmdForcePlatformType windows' ) +app_update ${SRCDS_APPID} $( [[ -z ${SRCDS_BETAID} ]] || printf %s \"-beta ${SRCDS_BETAID}\" ) $( [[ -z ${SRCDS_BETAPASS} ]] || printf %s \"-betapassword ${SRCDS_BETAPASS}\" ) ${INSTALL_FLAGS} validate +quit ## other flags may be needed depending on install. looking at you cs 1.6\r\n\r\n## set up 32 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk32\r\ncp -v linux32\/steamclient.so ..\/.steam\/sdk32\/steamclient.so\r\n\r\n## set up 64 bit libraries\r\nmkdir -p \/mnt\/server\/.steam\/sdk64\r\ncp -v linux64\/steamclient.so ..\/.steam\/sdk64\/steamclient.so\r\n\r\n## add below your custom commands if needed\r\n\r\n## install end\r\necho \"-----------------------------------------\"\r\necho \"Installation completed...\"\r\necho \"-----------------------------------------\"",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json]: same here apt update and apt install are not needed

File: game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json [hunk 5]
```diff
@@ -117,7 +127,8 @@
             "default_value": "CHANGEME",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json:127]: This should not have a default value

File: game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json [hunk 6]
```diff
@@ -153,7 +167,18 @@
...
+        {
+            "name": "SRCDS_APPID",
+            "description": "",
+            "env_variable": "SRCDS_APPID",
+            "default_value": "258550",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|max:20",
+            "field_type": "text"
         }
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json:182]: auto update variable?

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json (hunk 5 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json:127]: Not needed; check the used image

File: game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json (hunk 6 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json:182]: not needed; check used image

File: game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json (hunk 2 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json:137]: thats default in the egg; 

This PR is only change the images; if you want further changes, then add it after merging

File: game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json (hunk 1 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json]: its commented out. Better you import the Egg

File: game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json (hunk 3 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json:192]: not needed; chck used image

File: game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json (hunk 4 above)
Comment (gOOvER) [reply game_eggs/steamcmd_servers/rust/rust_staging/egg-rust-staging.json]: pls start importing the egg; the lines are commented out

**Review by QuintenQVD0 — APPROVED** [review]

**Comment by QuintenQVD0** [comment]
sorry , was a little blind thank you

**Pull request merged**